src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -239,18 +239,22 @@
            [category (update v :chats add-on-press)])
          categorized-chats)))
 
+
 (defn community-header
-  [name]
-  [quo/text
-   {:accessibility-label :chat-name-text
-    :number-of-lines     1
-    :ellipsize-mode      :tail
-    :weight              :semi-bold
-    :size                :heading-1
-    :style               {:margin-top (+ scroll-page.style/picture-radius
-                                         scroll-page.style/picture-border-width
-                                         12)}}
-   name])
+  [title logo]
+  [quo/text-combinations
+   {:container-style
+    {:margin-horizontal 0
```

Comment:
This style map is complicated enough that it should be in the `style` namespace.